apps/web/vibes/soul/sections/error/index.tsx

```diff
@@ -1,20 +1,23 @@
-import { ButtonLink } from '@/vibes/soul/primitives/button-link';
-
-interface Link {
-  label: string;
-  href: string;
-}
+import { Button } from '@/vibes/soul/primitives/button';
 
 interface Props {
   title?: string;
   subtitle?: string;
-  cta?: Link;
+  ctaLabel?: string;
+  ctaOnClick?: () => void;
+}
+
+function defaultOnClick() {
+  if (typeof window !== 'undefined') {
```

Comment:
There should be no need to check for `typeof window !== 'undefined'` since click handlers can only run in a environment with a DOM